frontend/src/containers/VerifyAccountPage/VerifyAccountPage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import './styles.css';
+import { withRouter } from 'react-router';
+import VerifyAccountCard from '../../components/VerifyAccountCard/VerifyAccountCard';
+import { verify } from '../../api/PublicAPI';
+import CustomSnackbar from '../../components/CustomSnackbar/CustomSnackbar';
+
+class VerifyAccountPage extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            showVerificationError: false,
+            showVerificationSuccess: false
+        }
+    }
+
+    componentDidMount() {
+        const { match } = this.props;
+        const id = match.params.userId;
+        verify(id).then(() => this.setState({ showVerificationSuccess: true }))
+            .catch(() => this.setState({ showVerificationError: true }))
+    }
+
+    render() {
+        const { showVerificationError, showVerificationSuccess } = this.state;
+
+        const hideVerificationError = () => {
+            this.setState({ showVerificationError: false })
+        }
+
+        const hideVerificationSuccess = () => {
+            this.setState({ showVerificationSuccess: false })
+        }
+
+        return (
+            <>
+                {showVerificationError &&
+                    <CustomSnackbar
+                        topCenter
+                        message="Error has occured during verification"
+                        onClose={hideVerificationError}
+                        severity="error" />
+                }
+                {showVerificationSuccess &&
```

Comment:
Doesn't `VerifyAccountCard` show the same message?

Reply:
You asked for this feature, man